rsts/getting_started.sh.rst

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+:'
+
+.. _gettingstarted:
+
+Getting started
+---------------
+
+.. rubric:: Estimated time to complete: 3 minutes.
+
+
+Prerequisites
+***************
+
+Make sure you have `docker installed <https://docs.docker.com/get-docker/>`__ and `git <https://git-scm.com/>`__ installed, then install flytekit:
+
```

Comment:
for sake of completion we can also have pip.

Reply:
agreed and python

Reply:
should we also specify the python version ?